lib/src/constants/config.dart

```diff
@@ -191,4 +192,8 @@ class AssetPickerConfig {
 
   /// {@macro wechat_assets_picker.PathNameBuilder}
   final PathNameBuilder<AssetPathEntity>? pathNameBuilder;
+
+  /// Whether the preview should auto play.
+  /// 预览是否自动播放
+  final bool shouldPreviewAutoPlay;
```

Comment:
I've checked all parameters in the config, and the verb should be placed in front of the name, so we should probably go with `shouldAutoplayPreview`.

Reply:
Agreed, `shouldPreviewAutoPlay` has updated to `shouldAutoplayPreview`.